### PR TITLE
add warning about renaming

### PR DESCRIPTION
**Pull request by rosieyohannan**
Fix for: https://circleci.atlassian.net/browse/CIRCLE-26385

### CODE REVIEW

File: jekyll/_cci2/gh-bb-integration.md [hunk 1]
```diff
@@ -65,6 +65,17 @@ Integrated status also appears on the pull request screen, to show that all test
     3. Regenerate the keys in CircleCI project.
 - Ensure no developer has access to a build in a repo with a User Key that requires more access than they have.
 
+## Renaming Orgs and Repositories
+
+If you find you need to rename an org or repo that you have previously hooked up to CircleCI, best practice is to follow these steps:
+
+1. Rename org/repo in VCS
+2. Head to the CircleCI application, using the new org name
```
Comment (rosieyohannan) [inline jekyll/_cci2/gh-bb-integration.md:73]: Can we give a format for the URL here? I guess `app.circleci.com/pipelines/<VCS>/<new-org-name>/<project-name>` from looking at my account, but maybe there's something simpler...?

Comment (emauton) [reply jekyll/_cci2/gh-bb-integration.md:73]: I believe that's the right URL to use.

File: jekyll/_cci2/gh-bb-integration.md [hunk 2]
```diff
@@ -65,6 +65,17 @@ Integrated status also appears on the pull request screen, to show that all test
...
+3. Confirm that your plan, projects and settings have been transferred successfully
+4. You are then free to create a new org/repo with the previously-used name in your VCS, if desired.
+
+**Note**: If these steps are not followed, it is possible, although rare that you may lose access to your org or repo settings, including **environment variables** and **contexts**.
```
Comment (rosieyohannan) [inline jekyll/_cci2/gh-bb-integration.md:77]:
```suggestion
**Note**: If these steps are not followed, it is possible, that you may lose access to your org or repo settings, including **environment variables** and **contexts**.
```

File: jekyll/_cci2/gh-bb-integration.md (hunk 1 above)
Comment (rosieyohannan) [inline jekyll/_cci2/gh-bb-integration.md:73]:
```suggestion
2. Head to the CircleCI application, using the new org/repo name, for example,  `app.circleci.com/pipelines/<VCS>/<new-org-name>/<project-name>`
```

File: jekyll/_cci2/gh-bb-integration.md [hunk 3]
```diff
@@ -65,6 +65,17 @@ Integrated status also appears on the pull request screen, to show that all test
     3. Regenerate the keys in CircleCI project.
 - Ensure no developer has access to a build in a repo with a User Key that requires more access than they have.
 
+## Renaming Orgs and Repositories
+
+If you find you need to rename an org or repo that you have previously hooked up to CircleCI, best practice is to follow these steps:
+
+1. Rename org/repo in VCS
+2. Head to the CircleCI application, using the new org/repo name, for example,  `app.circleci.com/pipelines/<VCS>/<new-org-name>/<project-name>`.
+3. Confirm that your plan, projects and settings have been transferred successfully
```
Comment (rosieyohannan) [inline jekyll/_cci2/gh-bb-integration.md:74]:
```suggestion
3. Confirm that your plan, projects and settings have been transferred successfully.
```

File: jekyll/_cci2/gh-bb-integration.md [hunk 4]
```diff
@@ -65,6 +65,17 @@ Integrated status also appears on the pull request screen, to show that all test
     3. Regenerate the keys in CircleCI project.
 - Ensure no developer has access to a build in a repo with a User Key that requires more access than they have.
 
+## Renaming Orgs and Repositories
+
+If you find you need to rename an org or repo that you have previously hooked up to CircleCI, best practice is to follow these steps:
+
+1. Rename org/repo in VCS
```
Comment (rosieyohannan) [inline jekyll/_cci2/gh-bb-integration.md:72]:
```suggestion
1. Rename org/repo in VCS.
```

**Pull request merged**